Add a configuration option for the executable path (#22)
fizruk/vscode-rzk: README.md

More examples:

![Syntax highlighting example.](images/example-relfunext2.png)

## Configuration

> Extension settings can be configured by going to the settings page (using the menu `File > Preferences > Settings`, or using the shortcut <kbd>CTRL + ,</kbd>).

Reviewer: ```suggestion
Extension settings can be configured by going to the settings page (using the menu `File > Preferences > Settings`, or using the shortcut <kbd>CTRL + ,</kbd> or <kbd>⌘ + ,</kbd>).
```

The currently available settings are:

| Name       | Type             | Default value | Description |
| ---------- | ---------------- | ------------- | ----------- |

> | `rzk.path` | `string \| null` | `null`        | The path to the `rzk` executable to use for the language server. Defaults to using the globally available one on the `PATH` |

Reviewer: ```suggestion
| `rzk.path` | `string \| null` | `null`        | The path to the `rzk` executable to use for the language server. `null` (default) means that `rzk` executable available in `PATH` will be used. |
```
